src/content/authors/lais.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Laís
+meta_title: "About Laís"
+image: /images/authors/lais.webp
```

Comment:
Your image and author information doesn't seem to show up still.
It should show your image and name next to the blog post date.
<img width="776" height="249" alt="Screenshot 2025-07-21 at 9 27 31 AM" src="https://github.com/user-attachments/assets/017a5d4b-11b7-48b7-adee-50c41eb28a57" />

Reply:
Fixed! Author profile renders successfully. Thanks for looking into it.